moveit_ros/perception/pointcloud_octomap_updater/src/pointcloud_octomap_updater.cpp

```diff
@@ -72,14 +72,14 @@ PointCloudOctomapUpdater::~PointCloudOctomapUpdater()
 
 bool PointCloudOctomapUpdater::setParams(const std::string& name_space)
 {
+  node_->get_parameter_or(name_space + ".ns", ns_, std::string());
```

Comment:
Please add a comment here so we can follow this logic in the future.

```suggestion
  // This parameter is optional
  node_->get_parameter_or(name_space + ".ns", ns_, std::string());
```